client/packages/system/src/Encounter/utils.ts

```diff
@@ -3,30 +3,32 @@ import { LocaleKey, TypedTFunction, useTranslation } from '@common/intl';
 import { EncounterNodeStatus } from '@common/types';
 import { EncounterRowFragment } from '@openmsupply-client/programs';
 
-const effectiveStatus = (
-  encounter: EncounterRowFragment,
+export const encounterStatusTranslation = (
+  status: EncounterNodeStatus,
   t: TypedTFunction<LocaleKey>
-) => {
-  const status = encounter.status;
-  if (!status) {
-    return '';
-  }
+): string => {
   switch (status) {
+    case EncounterNodeStatus.Pending:
+      return t('label.encounter-status-pending');
     case EncounterNodeStatus.Cancelled:
       return t('label.encounter-status-cancelled');
-    case EncounterNodeStatus.Completed:
-      return t('label.encounter-status-done');
-    case EncounterNodeStatus.Scheduled:
-      return t('label.encounter-status-scheduled');
-    case EncounterNodeStatus.Missed:
-      return t('label.encounter-status-missed');
+    case EncounterNodeStatus.Visited:
+      return t('label.encounter-status-visited');
     default:
-      ((_: never) => {
-        // exhaustive check
-        _;
-      })(status);
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      return ((_: never) => '')(status);
```

Comment:
Weird that this lint comment is required. The linter rules already has `'no-unused-vars': 0,`, very strange 🤔

Reply:
Okay, I figured it out with Mark -- there are two eslint rules that are in conflict:
```
    'no-unused-vars': 'off',
    '@typescript-eslint/no-unused-vars': 'error',
```

Just remove one of them and make sure the remaining one is set to "off" or `0`.

You can actually use a more specific setting  -- [argsIgnorePattern](https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern) to just ignore parameters starting with an underscore and still warn about the rest, which might be worth doing.

Reply:
Yes, have tested and this works:
```
 'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Just as long as you remove the typescript-specific rule as well.

Reply:
thanks! done that